Description: Disables APR library loading until we properly provide it.
Author: Thierry Carrez <thierry.carrez@ubuntu.com>
--- a/conf/server.xml
+++ b/conf/server.xml
@@ -24,7 +24,9 @@
   <Listener className="org.apache.catalina.security.SecurityListener" />
   -->
   <!--APR library loader. Documentation at /docs/apr.html -->
+  <!--
   <Listener className="org.apache.catalina.core.AprLifecycleListener" SSLEngine="on" />
+  -->
   <!--Initialize Jasper prior to webapps are loaded. Documentation at /docs/jasper-howto.html -->
   <Listener className="org.apache.catalina.core.JasperListener" />
   <!-- Prevent memory leaks due to use of particular java/javax APIs-->
